src/parser/open-api-v3/open-api-v3-parser.js

```diff
@@ -2,54 +2,113 @@ import refParser from 'json-schema-ref-parser';
 import getUIReadySchema from './schemaParser';
 
 /**
+ * Sort function
+ *
+ * @param {String} str1
+ * @param {String} str2
+ *
+ * @return {number}
+ */
+function sortByAlphabet(str1, str2) {
+  if (str1 < str2) {
+    return -1;
+  } else if (str1 > str2) {
+    return 1;
+  } else {
+    return 0;
+  }
+}
+
+/**
+ * Sort function
+ *
+ * @param {String} inMethod1
+ * @param {String} inMethod2
+ *
+ * @return {number}
+ */
+
+function defaultMethodSort(inMethod1, inMethod2) {
+  const methodWeights = {
```

Comment:
`methodWeights` can be extracted out as a constant for the project. Right now the variable will be created each time this function is called. Minor perf thing.